### PR TITLE
feat: remove facility from user profile

### PR DESCRIPTION
**Pull request by EugeneDevv**

### CODE REVIEW

**Comment by EugeneDevv** [comment]
Simulator Screen Shot - iPhone 13 Pro Max - 2022-10-24 at 08 51 37

File: lib/application/core/graphql/queries.dart [hunk 1]
```diff
@@ -167,8 +167,6 @@ query searchClientUser($searchParameter: String!) {
   searchClientUser(searchParameter: $searchParameter){
     ID
     CCCNumber
-    DefaultFacilityID
-    DefaultFacilityName
```
Comment (abiudrn) [inline lib/application/core/graphql/queries.dart:171]: Why was this deleted? It is useful for viewing the client's facility

File: lib/application/redux/actions/facilities/remove_facility_from_client_profile_action.dart [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    if (processedResponse.ok) {
+      final Map<String, dynamic> body = client.toMap(response);
+      client.close();
+
+      final String? errors = client.parseError(body);
+
+      if (errors != null) {
+        reportErrorToSentry(
+          hint: 'Error while removing facility from client',
```
Comment (abiudrn) [inline lib/application/redux/actions/facilities/remove_facility_from_client_profile_action.dart:64]: Extract these strings.

File: lib/application/redux/actions/facilities/remove_facility_from_client_profile_action.dart [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          query: removeFacilitiesFromClientProfileMutation,
+          response: response,
+          state: state,
+          variables: variables,
+        );
+      }
+    } else {
+      onFailure?.call(getErrorMessage('removing the facility.'));
+      reportErrorToSentry(
+        hint: 'Error while removing facility from client',
```
Comment (abiudrn) [inline lib/application/redux/actions/facilities/remove_facility_from_client_profile_action.dart:100]: Same here...

File: lib/application/redux/actions/facilities/remove_facility_from_staff_profile_action.dart [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+    if (processedResponse.ok) {
+      final Map<String, dynamic> body = client.toMap(response);
+      client.close();
+
+      final String? errors = client.parseError(body);
+
+      if (errors != null) {
+        reportErrorToSentry(
+          hint: 'Error while removing facility from staff',
```
Comment (abiudrn) [inline lib/application/redux/actions/facilities/remove_facility_from_staff_profile_action.dart:64]: Extract these hints into the hints folder

**Review by abiudrn — CHANGES_REQUESTED** [review]

File: lib/application/core/graphql/queries.dart (hunk 1 above)
Comment (EugeneDevv) [reply lib/application/core/graphql/queries.dart:171]: The API had changed, I am replacing them with the correct properties in my next PR
